bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_NLO_top_powheg_madspin/st_tch_4f_ckm_NLO_top_powheg_madspin_madspin_card.dat

```diff
@@ -0,0 +1,31 @@
+#************************************************************
```

Comment:
this file is called
`st_tch_4f_ckm_NLO_top_powheg_madspin_madspin_card.dat`
and should probably be called
`st_tch_4f_ckm_NLO_top_powheg_madspin_card.dat`
right?